ckanext/openafrica/templates/robots.txt

```diff
@@ -0,0 +1,78 @@
+# Crawl-delay Specifies the minimum interval (in seconds)
+#for a robot to wait after loading one page, before starting to load another.
+Crawl-delay: 10
+
+# Disallow specifies the paths that are not allowed to be crawled by the robot.
+User-agent: *
+Disallow: /dataset/rate/
+Disallow: /revision/
+Disallow: /dataset/*/history
+Disallow: /api/
+Disallow: /user/
+
+# Amazonbot
+User-agent: Amazonbot
```

Comment:
Comparing to other sites, we may be missing some crawlers
[The Guardian](https://www.theguardian.com/robots.txt), [Washington Post](https://www.washingtonpost.com/robots.txt), [BBC](https://www.bbc.com/robots.txt), etc.

Reply:
I believe this would be best addressed on [Roboshield](https://github.com/CodeForAfrica/ui/tree/main/apps/roboshield) so that we have a single source of truth for the `robots.txt`

Reply:
Referenced here https://github.com/CodeForAfrica/ui/issues/1052